packages/tools/client-debug-view/src/Debugger.tsx

```diff
@@ -32,16 +32,23 @@ export interface FluidClientDebuggerProps extends HasContainerId {
  * has been initialized, will display a note to the user and a refresh button to search again.
  */
 export function FluidClientDebugger(props: FluidClientDebuggerProps): React.ReactElement {
-	const { containerId } = props;
+	/**
+	 * Temporary hack until we support multiple containers / debuggers.
```

Comment:
I know I wrote this, so sorry to give feedback on it. But we might want to update this to explain what it's doing, in addition to why we need it.
```suggestion
	 * Queries the list of registered Client Debuggers, and returns the first one (arbitrarily) in the list.
	 * If there are no debuggers registered, will return `undefined`.
	 *
	 * @privateRemarks This is a temporary hack until we support multiple containers / debuggers.
```

Reply:
Update the comment